### DISCUSSION TITLE
[Question]: iOS can locally play HTTP stream from MPD?

### DISCUSSION THREAD

**Discussion by roiz-git** [Difegue/Stylophone#93] [Q&A] [answered]
I'm an android user planning carefully a migration to iOS. MPD has a big part of my life at home, work and even in my car.

Looking for an MPD client that can locally play HTTP stream coming from MPD server. Does Stylophone do that? If not, is it planned to be added? when?

Thanks

**Comment by Difegue** [comment, marked as answer]
Yes, local playback is supported on both Windows and iOS -- It'll require your server to have `httpd` as an output on port 8000.

**Reply by roiz-git** [reply]
yes, this is set already.

Sorry to bother you with this question, but could not find a documentation or a wiki

This is perfect!! 

Thanks!
